docker/edge/http/script.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#apt-get install jq -y
+
```

Comment:
[Major] I feel there should be a configuration file which should be read for below configuration rather than manually asking for the configuration. Generally configuration file is used in Fiware so same should be followed here.

Reply:
Added configuration file